### PR TITLE
Feature/scrum 78 functionality edit and delete button dashboard

### PR DESCRIPTION
**Pull request by Jackson201325**

### CODE REVIEW

File: app/api/wishList/[wishListId]/route.ts
Comment (Jackson201325) [inline app/api/wishList/[wishListId]/route.ts:1]: Se puede delete este creo ?

File: components/cards/gifts/components/add-to-wishlist-form.tsx [hunk 1]
```diff
@@ -41,8 +41,8 @@ function AddToWishListForm({ giftId, wishlistId }: AddToWishListFormProps) {
   return (
     // In the HTML form the action throw an error that the form was
     // submitted. Fix this is possible, does not seem to affect functionality
-    <form action={handleAddGiftToWishList} id={giftId}>
```
Comment (Jackson201325) [inline components/cards/gifts/components/add-to-wishlist-form.tsx:44]: Cambia de vuelta jejeje

Comment (Jackson201325) [reply components/cards/gifts/components/add-to-wishlist-form.tsx:44]: <form action={(formData) => handleAddGiftToWishList(formData)}

**Comment by lordalejandro** [comment]
the edit functionality still doesnt work, we leave this here and continue with the edit functinoality in a near future

**Pull request merged**